tests/letstest/multitester.py

```diff
@@ -402,42 +405,47 @@ def cleanup(cl_args, instances, targetlist):
 
 def main():
     # Fabric library controlled through global env parameters
-    env.key_filename = KEYFILE
-    env.shell = '/bin/bash -l -i -c'
-    env.connection_attempts = 5
-    env.timeout = 10
-    # replace default SystemExit thrown by fabric during trouble
-    class FabricException(Exception):
-        pass
-    env['abort_exception'] = FabricException
+    fab_config = Config(overrides={
+        "connect_kwargs": {
+            "key_filename": KEYFILE,
```

Comment:
```suggestion
            "key_filename": [KEYFILE],
```
If I try running these tests locally without this change, I hit https://github.com/fabric/fabric/issues/2007.

Reply:
thanks, weird that I didn't hit this!